src/Components/Ambulance/DriverDetailsForm.tsx

```diff
@@ -75,18 +75,18 @@ export const DriverDetailsForm = (props:any) => {
             err[key] = "Invalid phone number";
           }
           break;
-        case 'driverName2':
-          if (!value) {
-            err[key] = "This field is required"
-          }
-          break;
-        case 'cellNumber2':
-          if (!value) {
-            err[key] = "This field is required"
-          }else if(value && !(/^[0-9]{10}$/.test(form.cellNumber2))){
-            err[key] = "Invalid phone number";
-          }
-          break;
+        // case 'driverName2':
+        //   if (!value) {
+        //     err[key] = "This field is required"
+        //   }
+        //   break;
+        // case 'cellNumber2':
+        //   if (!value) {
+        //     err[key] = "This field is required"
+        //   }else if(value && !(/^[0-9]{10}$/.test(form.cellNumber2))){
+        //     err[key] = "Invalid phone number";
+        //   }
+        //   break;
```

Comment:
We should allow submit with blank t inputs, But i they add a value it should be validated.